ElementX/Sources/Services/Session/UserSession.swift

```diff
@@ -57,14 +57,19 @@ class UserSession: UserSessionProtocol {
         Task {
             switch await clientProxy.sessionVerificationControllerProxy() {
             case .success(let sessionVerificationController):
+                guard case let .success(isVerified) = await sessionVerificationController.isVerified() else {
+                    MXLog.error("Failed checking verification state. Will retry on the next sync update.")
+                    return
+                }
+                
                 tearDownSessionVerificationControllerWatchdog()
                 
-                if !sessionVerificationController.isVerified {
+                if isVerified == false {
```

Comment:
Would have thought some tool would pick this up if its not optional which I don't think it is.
```suggestion
                if !isVerified {
```